models/dnsrr.go

```diff
@@ -96,6 +96,8 @@ func RRtoRC(rr dns.RR, origin string) (RecordConfig, error) {
 		err = rc.SetTargetTLSA(v.Usage, v.Selector, v.MatchingType, v.Certificate)
 	case *dns.TXT:
 		err = rc.SetTargetTXTs(v.Txt)
+	case *dns.DHCID:
```

Comment:
Please keep this list alphabetized. (and if anything else is out of place, also move it)